util/libcrypto.num

```diff
@@ -4298,3 +4298,4 @@ UI_dup_user_data                        4240	1_1_1	EXIST::FUNCTION:UI
 UI_method_get_data_destructor           4241	1_1_1	EXIST::FUNCTION:UI
 ERR_load_strings_const                  4242	1_1_1	EXIST::FUNCTION:
 ASN1_TIME_to_tm                         4243	1_1_1	EXIST::FUNCTION:
+ASN1_TIME_set_string_X509               4244 	1_1_1	EXIST::FUNCTION:
```

Comment:
We have added some new functions in 1.1.0, but those were "things we forgot before the initial release and that really *should* have been there" or "we can't fix this bug without this function".

I'm not sure this PR is of either kind.